app/controllers/concerns/controller_exception_handling.rb

```diff
@@ -172,7 +172,7 @@ def template_for_html_error_response_page(page:)
   ##
   # Render a simple JSON error response
   def render_json_error_response(exception:, status:)
-    msg = Rack::Utils::HTTP_STATUS_CODES[status]
+    msg = Rack::Utils::HTTP_STATUS_CODES[status].dup
```

Comment:
Instead of doing this, could we add the frozen literal string magic comment to the file, which I reckon would have caught this, then revert this line and change the next to:
```ruby
     msg = "#{msg}: #{exception.message}" unless exception.message.blank?
```